src/content/docs/cheerp/12-reference/10-command-line-options/70-linear-heap-size.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "`-cheerp-linear-heap-size`"
+---
+
+`RuntimeError: memory access out of bounds` can occur if there is not enough heap memory. To increase the heap size, set `-cheerp-linear-heap-size=<value>` to a larger value (in MB, default is 1).
```

Comment:
The default value for `-cheerp-linear-heap-size` is **8**. Maybe you got this confused with `-cheerp-linear-stack-size`, which has a default value of 1.

Reply:
[Looks like this was wrong before](https://labs.leaningtech.com/cheerp/guides/interfacing-with-browser/WebAssembly-output#webassembly-heap-size) - will fix, thanks